src/ORM/EagerLoader.php

```diff
@@ -239,9 +239,7 @@ public function setMatching($assoc, callable $builder = null, $options = [])
 
         $pointer[$last] = ['queryBuilder' => $builder, 'matching' => true] + $options;
 
-        $this->_matching->contain($containments);
-
-        return $this;
+        return $this->_matching->contain($containments);
```

Comment:
Technically it doesn'the bteak BC has it is a new method. Weach could just document it properly in the migration guide. I think that is also a decent way of doing it.